test/metabase/api/search_test.clj

```diff
@@ -101,8 +118,8 @@
       search-item)))
 
 (defn- default-results-with-collection []
-  (on-search-types #{"dashboard" "pulse" "card" "dataset"}
-                   #(assoc % :collection {:id true, :name true :authority_level nil})
+  (on-search-types #{"dashboard" "pulse" "card" "dataset" "action"}
+                   #(assoc % :collection {:id true, :name (if (= (:model %) "action") nil true) :authority_level nil})
```

Comment:
We don't include `collection_name` in actions because it'd require another subselect and it doesn't get used. The reason it gets used for other models is we show `FooCard in BarCollection` in the search results. But for actions we just show `FooAction in BarModel`.

Reply:
Why does the client need the collection ID if it's not displayed in the search text? I think the collection id is needed for a link from the text.

e.g. 
<img width="273" alt="image" src="https://user-images.githubusercontent.com/39073188/217045308-2a16c43a-8798-4b4d-8326-8dde9190806a.png">

If the client doesn't need the collection id, then `action` should be removed from the set on line 119.

Reply:
Collection IDs are dealt with for actions for permissions reasons, and the search architecture isn't easily set up to later remove the ID from the results (afaik). But it didn't make sense to do extra work to include the collection name when that's not used anywhere. I'm not thrilled about needing this little `if` in the test helper, but I feel okay about it.

Reply:
> But it didn't make sense to do extra work

On second thought, the extra work is ~0, so why not.